tests/test_arrow.py

```diff
@@ -1954,7 +1954,7 @@ def test_years(self):
 
         result = arw.humanize(self.datetime)
 
-        assert result == "in 2 years"
+        assert result == "in a year"
 
```

Comment:
Should we adjust tests to account for the newly updated bounds? E.g. account for the edge cases?

Reply:
I can add some extra tests for edge cases.